EIPS/eip-5192.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 5192
+title: Minimal Soulbound NFTs
+description: Minimal standard interface for soulbinding EIP-721 NFTs
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-5192-minimal-soulbound-nfts/9814
+status: Review
+type: Standards Track
+category: ERC
+created: 2022-07-01
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [EIP-721](./eip-721.md). It proposes a minimal interface to make tokens soulbound using the feature detection functionality of [EIP-165](./eip-165.md).
+
+## Motivation
+
+The Ethereum community has expressed a need for non-transferrable, non-fungible, and socially-priced tokens similar to World of Warcraft’s soulbound items. But the lack of a token standard leads many developers to simply throw errors upon a user's invocation of transfer functionalities. Over the long term, this will lead to fragmentation and less composability.
+
+In this document, we outline a minimal addition to [EIP-721](./eip-721.md) that allows wallet implementers to check for a token contract's permanent (non-)transferrability using [EIP-165](./eip-165.md).
```

Comment:
How about adding EIP-1155, the other commonly-used NFT standard? The changes you suggest should work for both without needing modification.

Reply:
I don't care about 1155.